CRVResponse/src/CrvWidebandTest_module.cc

```diff
@@ -449,17 +449,20 @@ namespace mu2e
             }
           }
 
-          uint16_t offlineChannel = barIndex.asUint()*4 + SiPM;
-          CRVROC   onlineChannel  = crvChannelMap.online(offlineChannel);
-          uint16_t feb            = onlineChannel.FEB();
-          uint16_t febChannel     = onlineChannel.FEBchannel();
-
-          int index=feb*CRVId::nChanPerFEB+febChannel;
-          _recoPEs[index]   = recoPEs;
-          _recoTime[index]  = recoTime;
-          _fitStatus[index] = fitStatus;
-          _depositedEnergy[index] = depositedEnergy;
-          _histPEs[index]->Fill(recoPEs);
+          if(iSectorType==0)
```

Comment:
Please add the comment: "iSectorType==0 performs a track fit using all modules in the stack"

Reply:
I will make the change.